backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/TableOfContentsService.java

```diff
@@ -93,7 +96,45 @@ private TableOfContentsItem getTableOfContentsItem(final Element element) {
       ? List.of()
       : getChildren(element);
 
-    return new TableOfContentsItem(eId, marker, heading, type, children);
+    // eingebundene Stammform
+    var hasEingebundeneStammform = false;
+    var stammformEli = getEingebundeneStammformEli(element);
+    if (stammformEli.isPresent()) {
+      hasEingebundeneStammform = true;
+      heading = getHeadingForEingebundeneStammform(stammformEli.get()).orElse(null);
+    }
+
+    return new TableOfContentsItem(eId, marker, heading, type, hasEingebundeneStammform, children);
+  }
+
+  private Optional<DokumentEli> getEingebundeneStammformEli(Element element) {
+    return NodeParser.getElementFromExpression(
+      "componentRef[@showAs=\"regelungstext-eingebundene-stammform\"]",
+      element
+    ).map(contentRef -> {
+        var stammformEliRaw = contentRef.getAttribute("src");
+        // The src of an eingebundene Stammform has an extra file extension, which is not part of
+        // any type of ELI we support. Removing it to get a regular Dokument Expression ELI
```

Comment:
i think the DokumentManifestationEli supports it (even without the pointInTimeManifestation)